helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -969,3 +969,51 @@ opensearch:
       value: "true"
     - name: DISABLE_SECURITY_PLUGIN
       value: "true"
+alfresco-identity-adapter-service:
+  enabled: true
+  nameOverride: alfresco-identity-adapter-service
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
+      nginx.ingress.kubernetes.io/rewrite-target: /$1
+    path: "/identity-adapter-service/?(.*)"
+  image:
+    repository: quay.io/alfresco/alfresco-identity-adapter-service
+    tag: 7.5.0-alpha.1
+    pullPolicy: Always
+  liquibase:
+    enabled: false
+  postgresql:
+    enabled: false
+  probePath: /actuator/health
+  rabbitmq:
+    enabled: false
+  activiti:
+    keycloak:
```

Comment:
is there any triaging for the tasks about the removal of Keycloak refences from the infrastructure -like this one- and env vars?

Reply:
there is a separate issue for it, I prefer to don't misc stuff